pallets/chain-space/src/tests.rs

```diff
@@ -1082,3 +1083,22 @@ fn restoring_approval_of_a_space_by_non_root_should_fail() {
 		);
 	});
 }
+
+#[test]
+fn test_invalid_identifier_error() {
+	let invalid_base58_data = "I am not a Base58 encoded string".as_bytes().to_vec();
+	let valid_base58_data = [2u8; 30].to_vec();
```

Comment:
When this is changed to 40, which is still a valid length of identifier, still the error code "Invalid Identifier" is thrown. So this requires more investigation. 
Try with `let valid_base58_data = [2u8; 30].to_vec();`

Reply:
I investigated a bit and found something weird. I used the length 30 as it was used in the test `creating_a_invalid_identifier_length_should_fail` too. But here's the thing, if I change it to 40 even in the `creating_a_invalid_identifier_length_should_fail` test, it fails. I'm quite new to the code base so I'm not sure if that's supposed to happen. Would like to hear your thoughts on this.

Reply:
Yes that is what needs to be solved with #323. No errors should be propagated if identifier is within VALID length.

Reply:
Cool. Good effort and check there. Please send the PR.